tornjak-frontend/src/components/AccessNotAllowed.tsx

```diff
@@ -0,0 +1,9 @@
+const AccessNotAllowed = () => (
```

Comment:
Same with @maia-iyer comment above. This is just to handle a special case where a user knows the specific restricted link and tries to access it. Since there is no route available through the UI in viewer mode. It's not to access the API's associated with the route. That will give a 401.